app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(BaseException):

```

Comment:
The `NotWearingMaskError` should inherit from `Exception` instead of `BaseException`. This change is necessary to follow standard exception handling practices, as `BaseException` is intended for system-exiting exceptions.